From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Wed, 20 Oct 2021 22:19:25 +0200
Subject: dt-bindings: display: meson-vpu: Add the CVBS DAC properties

The CVBS DAC converts the digital video signal to the (analog) composite
video baseband signal (CVBS). This DAC is part of the HHI registers.
Add the phy and phy-names property to describe the relation between the
VPU (which outputs the digital signal) and the CVBS DAC.

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 Documentation/devicetree/bindings/display/amlogic,meson-vpu.yaml | 12 ++++++++++
 1 file changed, 12 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/amlogic,meson-vpu.yaml b/Documentation/devicetree/bindings/display/amlogic,meson-vpu.yaml
index 111111111111..222222222222 100644
--- a/Documentation/devicetree/bindings/display/amlogic,meson-vpu.yaml
+++ b/Documentation/devicetree/bindings/display/amlogic,meson-vpu.yaml
@@ -82,6 +82,15 @@ properties:
     description: should point to a canvas provider node
     $ref: /schemas/types.yaml#/definitions/phandle
 
+  phys:
+    maxItems: 1
+    description:
+      PHY specifier for the CVBS DAC
+
+  phy-names:
+    items:
+      - const: cvbs-dac
+
   power-domains:
     maxItems: 1
     description: phandle to the associated power domain
@@ -130,6 +139,9 @@ examples:
         #size-cells = <0>;
         amlogic,canvas = <&canvas>;
 
+        phys = <&cvbs_dac_phy>;
+        phy-names = "cvbs-dac";
+
         /* CVBS VDAC output port */
         port@0 {
             reg = <0>;
-- 
Armbian

